lib/gcpclient.js

```diff
@@ -38,7 +47,8 @@ GCPClient.prototype.authorize = function() {
  */
 GCPClient.prototype.showDeployDetail = function(userData, deployId) {
   var client = this;
-  return this.authorize().then(function(tokens) {
+  this.authorize().then(function(auth) {
```

Comment:
It looks like this works ok because I have the refresh token, but it causes Google to send a new access token more often than it needs to. I'm going to change it to save the access token if it changes to reduce the churn.